src/volseries.rs

```diff
@@ -296,8 +288,10 @@ impl<T, R, W, const S: usize> core::iter::DoubleEndedIterator
   #[inline]
   fn nth_back(&mut self, n: usize) -> Option<Self::Item> {
     if n < self.count {
+      let out =
+        Some(unsafe { self.base.cast::<[u8; S]>().add(1 + n).cast::<T>() });
```

Comment:
Shouldn't this be like the `VolBlockIter::nth_back` implementation?
```suggestion
        Some(unsafe { self.base.cast::<[u8; S]>().add(self.count - (n + 1)).cast::<T>() });
```

Reply:
Ah, yeah, I copied from one into the other to make sure the expression patterns matched and made them match too much.